openmsupply/open-msupply
# 1584 program requisitions program settings upsert/deletes

Fixes #1584 

# 👩🏻‍💻 What does this PR do? 
- Removes delete master list/master list line during sync
- Deletes all program requisition settings and order types during sync and regenerate
- Safe guard for reinitialisation if program id is missing from requisition

# 🧪 How has/should this change been tested? 
- Run normal tests
- Run integration tests
OR
- Create program setting/order types and editing them or deleting them on central
- Re-sync remote
- See new entries in database

## 💌 Any notes for the reviewer?


## 📃 Documentation

_No user facing changes_

or
